ppr-ui/src/components/mhrRegistration/HomeOwners/FractionalOwnership.vue

```diff
@@ -20,7 +20,7 @@
           :id="`fraction-amount-group-${groupId}`"
           label="Amount Owned by this Group"
           filled
-          v-model="fractionalData.interestNumerator"
+          v-model.number="fractionalData.interestNumerator"
```

Comment:
This is a good way to typecast 👍 
From the source of the value

Reply:
Yes, we need numbers here, but our validation is not robust enough to validate this, so I had a workaround (for now) by using two `isNumber` rules.